packages/react/src/vertex3.tsx

```diff
@@ -88,7 +91,7 @@ export const Vertex3: React.FunctionComponent<IVertex3> = ({
         fullAnnotationWidth += annotationGutter * (annotations.length - 1);
     }
     const textElement = <g transform={`translate(${textOffsetX} ${annoOffsetY})`}>
-        <TextBox
+        { text === "" ? null : <TextBox
```

Comment:
The "norm" seems to be to use "undefined" here instead of null (but suspect both work as expected)